frontend/rolecall/src/app/piece/piece_editor.component.ts

```diff
@@ -75,14 +85,31 @@ export class PieceEditor implements OnInit {
     this.pieceAPI.getAllPieces();
   }
 
+  private buildRenderingList() {
+    this.renderingItems = this.workingPieces.map((wpiece, wpieceIx) => {
+      const missingChidren = wpiece.type === 'SEGMENT' ? false : wpiece.positions.length === 0;
+      const prefix = missingChidren ? ' ' : 'z';
```

Comment:
`sortPrefix`, because it's not actually a prefix for the name

Reply:
Done.